packages/plugin-non-breaking-spaces/src/__tests__/index.test.ts

```diff
@@ -0,0 +1,95 @@
+import { Line } from "@lokse/core";
+
+import nonBreakingSpacesPlugin from "..";
+
+import Transformer from "../../../core/lib/transformer/json";
```

Comment:
Please don't break packages encapsulation 🙏 

```suggestion
import { transformersByFormat, OutputFormat } from "@lokse/core";

const Transformer = transformersByFormat[OutputFormat.JSON];
```